third-party/realdds/include/realdds/dds-discovery-sink.h

```diff
@@ -0,0 +1,39 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
```

Comment:
2024. Can also update in other changed files.

Reply:
I'll update in the new file :)